README.md

```diff
@@ -62,14 +67,25 @@ For version that works with Spark 1.x, please check for the [1.x branch](https:/
 
 Currently, only master-SNAPSHOT is supported.
 
+NOTE: In the examples below, `2.11` is the Scala version. If you are using a different version, be sure to update these values accordingly.
+
 ### Release builds
 You may use this library in your applications with the following dependency information:
 
+- **Without build tool**:
+    ```bash
+    spark-submit \
+      --deploy-mode cluster \
+      --master yarn \
+      --jars https://s3.amazonaws.com/redshift-downloads/drivers/jdbc/1.2.36.1060/RedshiftJDBC42-no-awssdk-1.2.36.1060.jar\
+      --packages org.apache.spark:spark-avro_2.11:2.4.2,io.github.spark-redshift-community:spark-redshift_2.11:4.0.0 \
```

Comment:
I'm not sure why `spark-avro` needs to be specified, when it's declared as a dependency in `build.sbt`? But it does, at least on EMR 5.24.